Gemfile

```diff
@@ -12,9 +12,14 @@ gem "faker"
 # Git. Remember to move these dependencies to your gemspec before releasing
 # your gem to rubygems.org.
 
-# To use a debugger
-# gem 'byebug', group: [:development, :test]
+gem 'byebug', group: [:development, :test]
 
 group :test do
   gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
+  gem 'factory_bot_rails', '~> 6.2'
+  gem 'shoulda', '~> 4.0'
+  gem 'database_cleaner', '~> 2.0', '>= 2.0.1'
+  gem 'rspec-rails', '~> 5.0', '>= 5.0.2'
+  gem 'vcr', '~> 6.1'
```

Comment:
Nao acho que precisamos usar o VCR para testar aqui... ACho melhor nao fazer

Reply:
Na minha visao nao precisa fazer isso para as requests externas... da pra usar um mock simples...

Como isso aqui é uma gem, nao podemos encher de depdendencias

Reply:
mas sera que da BO sendo que vamos deixar só como dependência pra teste, quem for instalar pra usar nao deve instalar essas gens mesmo